nixos/modules/virtualisation/libvirtd.nix

```diff
@@ -293,7 +293,7 @@ in
         # Copy default libvirt network config .xml files to /var/lib
         # Files modified by the user will not be overwritten
         for i in $(cd ${cfg.package}/var/lib && echo \
-            libvirt/qemu/networks/*.xml libvirt/qemu/networks/autostart/*.xml \
```

Comment:
We could just check if `libvirt/qemu/networks/autostart/` exists and if it does not, create and prepopulate it with the link to the default network. This would fix the initial setup.

Reply:
I see your point and I do agree. Thanks. :)